pkg/networkservice/common/excludedprefixes/utils.go

```diff
@@ -31,3 +32,20 @@ func removeDuplicates(elements []string) []string {
 	}
 	return result
 }
+
+// Exclude - excludes strings from a slice
+func Exclude(source, exclude []string) []string {
```

Comment:
Why is this public?

Reply:
This is used in tests which is in separate package. Shall we move this to `tools`?

Reply:
Removed